jwst/combine_1d/combine1d.py

```diff
@@ -341,7 +341,15 @@ def count_input(input_spectra):
 
     # Create an array with all the input wavelengths (i.e. the union
     # of the input wavelengths).
-    wl = np.hstack([in_spec.wavelength for in_spec in input_spectra])
+    wl = None
+    for in_spec in input_spectra:
+        input_wl = in_spec.wavelength
+        # only include spectra that have more than 1 data point
+        if len(input_wl) > 1:
+            if wl is None:
+                wl = input_wl
+            else:
+                wl = np.hstack((input_wl, wl))
     wl.sort()
```

Comment:
If `len(input_wl) == 1` for all `in_spec`, then `wl` will be `NoneType` and will error out when trying to sort.  Is it possible for no spectra to have more than one data point?

Reply:
I guess in theory anything is possible! But if that happens to be the case, it would indicate that we have even bigger problems somewhere upstream and would like to see an actual error occur in order to bring it to our attention.